dts/arm/infineon/psoc6/psoc6_02/psoc6_02.124-bga.dtsi

```diff
@@ -0,0 +1,2051 @@
+/*
+ * Copyright (c) 2021, Cypress Semiconductor
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/gpio/gpio.h>
+#include "psoc6_02.dtsi"
+#include "dt-bindings/pinctrl/cat1-pinctrl.h"
```

Comment:
please don't use `""` for non-local includes

Reply:
updated